.github/workflows/tagged_master_release.yml

```diff
@@ -111,7 +111,7 @@ jobs:
           cpack -D CPACK_PACKAGE_FILE_NAME=${{ matrix.artifact }}
         shell: cmd
       - name: Upload windows artifact
-        if: matrix.os == 'windows-latest'
+        if: matrix.os == 'windows-2019'
```

Comment:
not blocking but it seem like you could make this a variable you can reference instead of having to change this so many places

Reply:
Sorry, this is noise you can ignore because it should stabilize once tensorstore is settled again. It's just me thrashing on it.  This should not be changing this often.